diff --git a/ios/RNExitApp/RNExitApp.h b/ios/RNExitApp/RNExitApp.h
index 50a21358db675bafd465874766ff13e753376f5e..572fa91df7f358fcc1d2f619c03fb241a6e8ecad 100644
--- a/ios/RNExitApp/RNExitApp.h
+++ b/ios/RNExitApp/RNExitApp.h
@@ -7,7 +7,7 @@
 #endif
 
 #if RCT_NEW_ARCH_ENABLED
-#import <React-Codegen/RNExitAppSpec/RNExitAppSpec.h>
+#import  "RNExitAppSpec.h"
 #endif
 
 @interface RNExitApp : NSObject <RCTBridgeModule>
diff --git a/ios/RNExitApp/RNExitApp.mm b/ios/RNExitApp/RNExitApp.mm
index bec59482914b435e95886dd6da30fe627ad4fc8d..020c05a5810fb2541820b8dd568254dba67d8756 100644
--- a/ios/RNExitApp/RNExitApp.mm
+++ b/ios/RNExitApp/RNExitApp.mm
@@ -3,7 +3,7 @@
 #import "RNExitApp.h"
 
 #if RCT_NEW_ARCH_ENABLED
-#import <RNExitAppSpec/RNExitAppSpec.h>
+#import  "RNExitAppSpec.h"
 #endif
 
 @implementation RNExitApp
